File: ASMR.Web/ClientApp/src/data/release/ASMRMobileReleaseInformation.ts
import AndroidReleaseInformation from "@asmr/data/release/common/AndroidReleaseInformation"
import iOSReleaseInformation from "@asmr/data/release/common/iOSReleaseInformation"

interface ASMRMobileReleaseInformation {
	Android: AndroidReleaseInformation
	iOS: iOSReleaseInformation
}

export default ASMRMobileReleaseInformation

File: ASMR.Web/ReleaseInformation/ASMRMobileReleaseInformation.cs
//
// asmr: Coffee Beans Management Solution
// © 2021 Pandora Karya Digital. All right reserved.
//
// Written by Danang Galuh Tegar Prasetyo [connect@danang.id]
//
// ASMRMobileReleaseInformation.cs
//

using System.Text.Json.Serialization;
using ASMR.Web.ReleaseInformation.Common;

// ReSharper disable InconsistentNaming
namespace ASMR.Web.ReleaseInformation;

public class ASMRMobileReleaseInformation
{
	[JsonPropertyName("Android")] public AndroidReleaseInformation Android { get; set; }

	[JsonPropertyName("iOS")] public iOSReleaseInformation iOS { get; set; }
}